src/screens/FeedScreen/hooks/useFeed.js

```diff
@@ -97,13 +96,13 @@ const useFeed = () => {
 
   const handleTextCountStyle = () => {
     if (totalVote > 0) {
-      return COLORS.holyTosca;
+      return '#00ADB5';
     }
     if (totalVote < 0) {
-      return COLORS.red;
+      return '#FF2E63';
     }
 
-    return COLORS.gray1;
+    return '#C4C4C4';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The color codes used here are hardcoded. It's a good practice to define these colors in a centralized place and then import them where needed. This makes it easier to manage and update the color scheme of your application.

```diff
-      return '#00ADB5';
+      return colors.holyTosca;
...
-      return '#FF2E63';
+      return colors.red;
...
-     return '#C4C4C4';
+     return colors.gray1;
```
Please ensure that the `colors` object is imported at the top of the file and contains the appropriate color definitions.


<!-- This is an auto-generated comment by OSS CodeRabbit -->